src/iiiftilesource.js

```diff
@@ -444,11 +444,10 @@ $.extend( $.IIIFTileSource.prototype, $.TileSource.prototype, /** @lends OpenSea
     /**
      * Determine whether arbitrary tile requests can be made against a service with the given profile
      * @function
-     * @param {array} options - IIIF profile array
-     * @param {Array|String} options.profile
+     * @param {Object} options - IIIF option
```

Comment:
I feel like even saying "IIIF option" here implies something that it's not... these are just some of the possible IIIF options but not all of them. I think it's best to just leave the " - IIIF option" off so the line just reads `* @param {Object} options`.